File: WebBiblioteca_Api/BibliotecaApiDLL/Autor/Autor.cs
namespace BibliotecaApiDLL.Autor
{
    public class Autor
    {
        public Autor(int codAutor, string nome, string descricao)
        {
            CodAutor = codAutor;
            Nome = nome;
            Descricao = descricao;
        }

        public int CodAutor { get ; set; }
        public string Nome { get; set; }
        public string Descricao { get; set; }
    }
}

File: WebBibliotecaApp/src/Objects/Autor.ts
export class Autor {
    codAutor: number;
    nome: string = '';
    descricao: string = '';

    constructor (codAutor: number, nome: string, descricao: string){
        this.codAutor = codAutor;
        this.descricao = descricao;
        this.nome = nome;
    }
}
